channels.go

```diff
@@ -0,0 +1,56 @@
+package helix
+
+/*
+This file makes up for a missing API in Helix. This appears to be a recent
+change (as of around 2020-08-03) to the Twitch API, where GetStreams no longer
+returns information about offline channels.
+*/
+
+// SearchChannelsParams is parameters for SearchChannels
+type SearchChannelsParams struct {
+	Channel  string `query:"query"`
+	After    string `query:"after"`
+	First    int    `query:"first,20"` // Limit 100
+	LiveOnly bool   `query:"live_only"`
+}
+
+// ManySearchChannels is the response data from SearchChannels
+type ManySearchChannels struct {
+	Channels   []Channel  `json:"data"`
+	Pagination Pagination `json:"pagination"`
+}
+
+// Channel describes a channel from SearchChannel
+type Channel struct {
+	ID           string `json:"id"`
+	GameID       string `json:"game_id"`
+	DisplayName  string `json:"display_name"`
+	Language     string `json:"broadcaster_language"`
+	Title        string `json:"title"`
+	ThumbnailURL string `json:"thumbnail_url"`
+	IsLive       bool   `json:"is_live"`
+	// StartedAt    time.Time `json:"started_at"` // This can be invalid if the channel is not live; omitting it. Prefer GetStreams for live channels.
```

Comment:
Can I suggest the following?

```suggestion
	StartedAt    Time `json:"started_at"`
```

This library ships with its own `Time` struct that should handle cases such as these.

See https://github.com/nicklaw5/helix/blob/master/time.go#L21-L37

Reply:
I will change this. FYI, I based this struct on Stream in streams.go, which is using time.Time.